src/PathMaster.ts

```diff
@@ -9,7 +9,7 @@ import { Config } from "./Config";
  * If a import url isn't relative
  * and only has ascii + @ in the name it is considered a node module
  */
-const NODE_MODULE = /^([a-z@].*)$/;
+const NODE_MODULE = /^[^:]([a-z@].*)$/;
```

Comment:
fwiw official package name tester : https://github.com/npm/validate-npm-package-name but that has more validation than what you probably want : https://github.com/npm/validate-npm-package-name/blob/ddf73c79e920b59413485ca2ae50a38552156d62/index.js :rose: